devex/cross-nuget/windows/build.ps1

```diff
@@ -0,0 +1,38 @@
+﻿# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Build for SGX
+# |- Default (with LVI)
+#    |- Debug
+#    |- Release
+
+$ErrorActionPreference = "Stop"
+
+If (-not (Test-Path -Path SDK))
+{
+    git clone --recursive --depth=1 https://github.com/openenclave/openenclave SDK -b v0.11.0
```

Comment:
Can "v0.11.0" be put into a powershell variable to something to make it easier to pick out?

Reply:
Indeed, done.